code/modules/recycling/disposal.dm

```diff
@@ -464,7 +464,7 @@
 		H.tomail = 1
 
 	sleep(10)
-	if(last_sound < world.time + 1)
+	if(last_sound + 0.1 SECONDS < world.time)
```

Comment:
same here and on L502

Reply:
Made this one a global define as we have to use it in two files